lottie-swift/src/Public/MacOS/BundleImageProvider.swift

```diff
@@ -34,11 +34,17 @@ public class BundleImageProvider: AnimationImageProvider {
   
   public func imageForAsset(asset: ImageAsset) -> CGImage? {
     
-    if asset.name.hasPrefix("data:"),
-      let url = URL(string: asset.name),
-      let data = try? Data(contentsOf: url),
-      let image = NSImage(data: data) {
-      return image.CGImage
+    if asset.name.hasPrefix("data:") {
```

Comment:
A comment about why we can't use the default `UIImage` / `NSImage` initializer (linking to #1256) would be helpful